Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -85,6 +85,9 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
     open lazy var router = components
         .messageListRouter
         .init(rootViewController: self)
+
+    /// The height of the typing indicator view
+    open var typingIndicatorViewHeight: CGFloat = 22
```

Comment:
This should be `private(set)` since the value can't be changed dynamically

Reply:
Done ✅